pkg/api/v1/types.go

```diff
@@ -1983,6 +1987,12 @@ type NodeAffinityPreset struct {
 	Values []string
 }
 
+type TopologySpreadConstraint struct {
+	MaxSkew           string `json:"maxSkew"`
```

Comment:
Should not we have this type as an integer ? or at least check that we have an integer value set  (and add a unit-test then)?
That would prevent from creating a MD with a non-integer value.